crates/common/types/account.rs

```diff
@@ -19,11 +19,60 @@ use crate::{
     utils::keccak,
 };
 
-#[allow(unused)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Code {
+    pub hash: H256,
+    pub bytecode: Bytes,
+    // TODO: Consider using Arc<[u16]> (needs to enable serde rc feature)
+    pub jump_targets: Vec<u16>,
```

Comment:
This needs a comment explaining the reason for using `u16` instead of bigger numbers. Linking to https://eips.ethereum.org/EIPS/eip-170 and https://eips.ethereum.org/EIPS/eip-7907. Also, EIP-7907 might break this since it bumps initcode max size to 96Kb.